tests/modules/withenum.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
yes, I find files easier to read that way (maybe because it just adds room between my IDE's menu bar and the first line of code). It may not be a proper habit or a recommended one :shrug: